views/all_countries.erb

```diff
@@ -1,11 +1,13 @@
 <div class="page-section" id="home">
     <div class="container">
         <ul class="grid-list grid-list--vertically-center">
-          <% @page.world.sort_by { |k, _| k }.each do |k, v| %>
+          <% @page.world.as_json.sort_by { |key, _| key }.each do |key, value| %>
```

Comment:
The goal here is to kill that `as_json` method. You'll probably want to pass `world.all_countries` or equivalent into the template which provides a list of country objects.